sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -117,9 +122,9 @@
      */
     private AnalyzeSentimentResult convertToTextSentimentResult(final DocumentSentiment documentSentiment) {
```

Comment:
```suggestion
    private AnalyzeSentimentResult convertToSentenceSentimentResult(final DocumentSentiment documentSentiment) {
```

Reply:
This method convert a `DocumentSentiment` to `AnalyzeSentimentResult`. I will change to `convertToAnalyzeSentimentResult`